src/Resources/Locales/de_DE.axaml

```diff
@@ -454,7 +455,7 @@
   <x:String x:Key="Text.RemoteCM.Edit" xml:space="preserve">Bearbeiten...</x:String>
   <x:String x:Key="Text.RemoteCM.Fetch" xml:space="preserve">Fetch</x:String>
   <x:String x:Key="Text.RemoteCM.OpenInBrowser" xml:space="preserve">Im Browser öffnen</x:String>
-  <x:String x:Key="Text.RemoteCM.Prune" xml:space="preserve">Aufräumen (Prune)</x:String>
+  <x:String x:Key="Text.RemoteCM.Prune" xml:space="preserve">Prune</x:String>
```

Comment:
"Aufräumen" is good translation, but there are multiple other places where we then need to translate "Prune". I am fine with doing that but I just went for "Prune" consistently instead.